Orange/widgets/data/oweditdomain.py

```diff
@@ -916,13 +1092,12 @@ def __init__(self, *args, **kwargs):
             objectName="action-merge-item",
             toolTip="Merge selected items.",
             shortcut=QKeySequence(Qt.ControlModifier | Qt.Key_Equal),
-            shortcutContext=Qt.WidgetShortcut,
-            enabled=False,
+            shortcutContext=Qt.WidgetShortcut
         )
 
         self.add_new_item.triggered.connect(self._add_category)
         self.remove_item.triggered.connect(self._remove_category)
-        self.merge_items.triggered.connect(self._merge_categories)
+        self.merge_items.triggered.connect(self._merge_less_common_categories)
```

Comment:
Given that it also merges selected categories, the old name, `_merge_categories`, fits better.

Reply:
Agree. It is was changed since first I taught there will be two dialogues. It is fixed now.